portfolio/src/main/webapp/script.js

```diff
@@ -47,6 +47,9 @@ function randomFood() {
     foodHolder.appendChild(imgFood);
 }
 
+/** 
+ * Retrives the text on the .../data URL and adds it to the DOM
+ */
 function getData() {
   fetch('/data').then(response => response.text()).then((quote) => {
     document.getElementById('data-container').innerHTML = quote;
```

Comment:
// No change needed.

I think you asked at the top of this PR about what the instructions were suggesting for the client.

So, one of the benefits of using the JSON format when sending down the list of comments is that Javascript can read JSON text as a structured object. In this case that means that in javascript can treat the 'quote' variable like a list. So, for instance, if you wanted to put each message in it's own <li> tag, you could use a for loop to go through the elements in 'quote' one at a time and append each within it's own <li> tag.

You don't need to do so in this PR but if you have time later this week I'd encourage you to update the client to use this structure and output the messages in whatever html format you prefer